ribs/visualize.py

```diff
@@ -497,12 +518,13 @@ def parallel_axes_plot(archive,
         archive (ArchiveBase): Any form of archive.
         ax (matplotlib.axes.Axes): Axes on which to plot the parallel 
             coordinates plot. If None,the current axis will be used.
-        bc_order (list of int or list of tuples): By default, the BCs of the 
-            parallel axes will appear in-order. To switch the axes around, 
-            supply a list of the
-            behavior axes (e.g.,``['behavior_1', 'behavior_0', 'behavior_2']``). 
-            It is also possible to plot less behaviors than the number of BCs in
-            the archive (the example also works for archives of 4 BCs).
+        bc_order (list of int or list of tuples): if a list of ints, this
```

Comment:
I was originally thinking it could be a list of (int or tuple) e.g. [1, (1,"a"), ...]. But this makes more sense -- I can't see why you would want to name some dimensions but not others.